nettests/websites/web_connectivity.go

```diff
@@ -1,10 +1,56 @@
 package websites
 
 import (
+	"encoding/json"
+	"fmt"
+	"io/ioutil"
+	"net/http"
+
 	"github.com/measurement-kit/go-measurement-kit"
 	"github.com/openobservatory/gooni/nettests"
+	"github.com/pkg/errors"
 )
 
+type URLInfo struct {
+	URL          string `json:"url"`
+	CategoryCode string `json:"category_code"`
+}
+
+type URLResponse struct {
+	Results []URLInfo `json:"results"`
+}
+
+const orchestrateBaseURL = "https://events.proteus.test.ooni.io"
+
+func lookupURLs(ctl *nettests.Controller) ([]string, error) {
+	var (
+		parsed = new(URLResponse)
+		urls   []string
+	)
+	reqURL := fmt.Sprintf("%s/api/v1/urls?probe_cc=%s",
+		orchestrateBaseURL,
+		ctl.Ctx.Location.CountryCode)
+
+	resp, err := http.Get(reqURL)
```

Comment:
should we be able to request this through tor / domain fronting? should we have a central request manager interface to deal with that?

Reply:
Yes for sure. This is just quick and dirty prototype code.

Reply:
I should also add, that this functionality should actually be part of measurement-kit (that of fetching URL lists from the orchestration backend), however I implemented it inside of gooni for the time being to get the whole prototype working faster.